arangod/Pregel/Worker/State.h

```diff
@@ -95,6 +100,7 @@ struct WorkerState {
   uint32_t messageBatchSize = 500;
```

Comment:
It would be really helpful to extract as many of these variables into the Computing state as possible, my guess would be that you can move isWaitingForAllMessagesSince, readCache, writeCache and messageBatchSize into the Computing state.

Reply:
Cannot move `readCache` and `writeCache` since they're initialized in the `WorkerState` constructor, based on the `messageCombiner` field.

That leaves just `isWaitingForAllMessagesSince` and `messageBatchSize` as candidates for moving into the `Computing` state. Not sure if the refactor makes sense for just two fields.